### PR TITLE
Add cleaninstall task

### PR DESCRIPTION
**Pull request by NotAShelf**
#### Describe your PR, what does it fix/add?
Adds a cleaninstall option in the Makefile, this is so that users that launch Hyprland through a wrapper on login managers don't have to update their hyprland.desktop file everytime they build Hyprland from source. It's a clone of the install task without the part that copies the example desktop file.

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
Maybe cleaninstall could be something else, but nothing breaking.

#### Is it ready for merging, or does it need work?
Ready for merging.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -137,6 +137,25 @@ install:
 	cp ./assets/wall_8K.png ${PREFIX}/share/hyprland
 
 	install -Dm644 -t ${PREFIX}/share/man/man1 ./docs/*.1
+	
+cleaninstall:
```
Comment (vaxerski) [inline Makefile:141]: why not
```
make clear
make install
```
?

Comment (NotAShelf) [reply Makefile:141]: correct me if I'm wrong, but `make install` still copies the desktop file to /usr/share/wayland-sessions

and also, I wanted this task to be global, functional regardless of it's an old installation or freshly built from source

Comment (vaxerski) [reply Makefile:141]: oh right yea mb

**Comment by vaxerski** [comment]
update the wiki plox

**Pull request merged**